language/stdlib/modules/libra_coin.move

```diff
@@ -15,20 +15,33 @@ module LibraCoin {
     resource struct MarketCap {
         // The sum of the values of all LibraCoin::T resources in the system
         total_value: u128,
+        // A holding area where funds that will subsequently be burned wait while their underyling
+        // assets are sold off-chain
+        preburn: T,
     }
 
-    // Return a reference to the MintCapability published under the sender's account. Fails if the
-    // sender does not have a MintCapability.
-    // Since only the Association account has a mint capability, this will only succeed if it is
-    // invoked by a transaction sent by that account.
-    public fun mint_with_default_capability(amount: u64): T acquires MintCapability, MarketCap {
+    // Return `amount` coins created using the MintCapability published under the sender's account.
+    // Fails if the sender does not have a published MintCapability.
+    public fun mint_with_capability(amount: u64): T acquires MintCapability, MarketCap {
```

Comment:
This "mint_with_capability" name seems wrong to me. Compare to some other functions:

- deposit_with_metadata: similar to deposit() but with an extra metadata argument
- deposit_with_sender_and_metadata: similar to deposit_with_metadata() but with the sender argument as well
- withdraw_with_capability: similar to other withdraw functions but with an explicit WithdrawalCapability argument
- rotate_authentication_key_with_capability: similar to rotate_authentication_key but with an explicit KeyRotationCapability argument

For consistency, we should rename mint() to mint_with_capability and then mint_with_default_capability() can be just plain "mint".

Reply:
Good calls, will fix.